sdcm/cluster.py

```diff
@@ -3531,7 +3531,8 @@ def verify_node_setup(start_time):
                 setup_thread = threading.Thread(target=node_setup, name='NodeSetupThread',
                                                 args=(node,), daemon=True)
                 setup_thread.start()
-                time.sleep(120)
+                if isinstance(cl_inst, BaseScyllaCluster):
+                    time.sleep(120)
```

Comment:
Please, add a log message that we're waiting for 2 minutes in this place.

Reply:
@enaydanov Done.